app/models/product.model.js

```diff
@@ -9,17 +9,17 @@ const ProductSchema = mongoose.Schema(
     },
     price: Number,
     color: String,
+    cart: String,
     imageUrl: String,
     description: String,
     categories: {
       type: String,
-      enum: ["accesories", "building", "outfits",]
+      enum: ["Accessories", "Building", "Outfits", "Please note that this item might be available based on the delivery area"]
```

Comment:
what's the meaning of this `Please note that this item might be available based on the delivery area..`